src/sensors/heading/gyroscope/GY50.cpp

```diff
@@ -64,24 +61,25 @@ void GY50::attach()
         return;
     }
 
+    static const uint8_t controlRegister1 = 0x20;
+    static const uint8_t controlRegister2 = 0x21;
+    static const uint8_t controlRegister3 = 0x22;
+    static const uint8_t controlRegister4 = 0x23;
+    static const uint8_t controlRegister5 = 0x24;
+
     mRuntime.i2cInit();
     // Enable z and turn off power down
-    static constexpr uint8_t kControlRegister1 = 0x20;
-    writeL3G4200DRegister(kControlRegister1, 0b00001100); // NOLINT(readability-magic-numbers)
+    writeL3G4200DRegister(controlRegister1, 0b00001100);
```

Comment:
:warning: **readability-magic-numbers** :warning:
0b00001100 is a magic number; consider replacing it with a named constant